config/pmd/ruleset.xml

```diff
@@ -99,5 +100,10 @@
       <property name="maxfields" value="50" />
     </properties>
   </rule>
+  <rule ref="category/java/design.xml/TooManyMethods">
```

Comment:
any specific reason for adding this rule?

Reply:
In some test class too many test methods we need, but when do so, it gives pmd error. to avoid that, I have added the line